Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -52,7 +53,8 @@ const Contest = () => {
         },
     } = useProblems();
     const { actions: { changePreviousProblemSubmissionsPage } } = useProblemSubmissions();
-    const { state: { user: { permissions: { canAccessAdministration } } } } = useAuth();
+    const { internalUser } =
```

Comment:
Maybe just rename this to user as its not an internal variable since its used in Contest?

Reply:
Fixed